### PR TITLE
Mature execute_query API

### PR DESCRIPTION
**Pull request by robsdedude**
* Renaming `neo4j.RoutingControl.WRITERS` to `WRITE` and `READERS` to `READ`.
 * Renaming `driver.query_bookmark_manager()` to `execute_query_bookmark_manager()`.
 * Remove experimental tag from execute_query and its related APIs
   * `driver.execute_query`
   * `driver.execute_query_bookmark_manager`
   * `AsyncBookmarkManager`, `BookmarkManager`, the corresponding config option (session's `bookmark_manager`) and factory method (`driver.bookmark_manager()`)
   * `RoutingControl` enum
   * `EagerResult` and `result.to_eager_result()`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 See also https://github.com/neo4j/neo4j-python-driver/wiki for more details.
 
 ## NEXT RELEASE
-- ...
+- Renamed experimental `neo4j.RoutingControl.READERS` to `READ` and `WRITERS` to `WRITE`.
+- Renamed experimental `driver.query_bookmark_manager` to `execute_query_bookmark_manager`.
```
Comment (bigmontz) [inline CHANGELOG.md:8]: Should you also add here statements about the apis which are not exp anymore?

File: tests/unit/async_/test_driver.py [hunk 2]
```diff
@@ -316,16 +297,14 @@ async def test_get_server_info_parameters_are_experimental(
 
 @mark_async_test
 async def test_with_builtin_bookmark_manager(mocker) -> None:
-    with pytest.warns(ExperimentalWarning, match="bookmark manager"):
-        bmm = AsyncGraphDatabase.bookmark_manager()
+    bmm = AsyncGraphDatabase.bookmark_manager()
```
Comment (bigmontz) [inline tests/unit/async_/test_driver.py:300]: Where are the changes in the `bookmark_manager` to promote it to GA?

File: CHANGELOG.md (hunk 1 above)
Comment (robsdedude) [reply CHANGELOG.md:8]: This changelog only contains breaking and major changes (see line 1).

File: tests/unit/async_/test_driver.py (hunk 2 above)
Comment (robsdedude) [reply tests/unit/async_/test_driver.py:300]: The interface: [src/neo4j/api.py](https://github.com/neo4j/neo4j-python-driver/pull/911/files#diff-3c42f876afff664c25f7b8272bbb9751ce79ed419eaceddcab463298d8d1e67b)
The factory method: [src/neo4j/_async/driver.py](https://github.com/neo4j/neo4j-python-driver/pull/911/files#diff-de8642ef49863ea0a16262621b7059131e825d0387a3f261d7249f4d7d8d5b16)

Comment (bigmontz) [reply tests/unit/async_/test_driver.py:300]: I've mixed with the `execute_query_boomark_manager`.

**Review by bigmontz — APPROVED** [review]
🔖 🕴🏼

**Review by thelonelyvulpes — APPROVED** [review]

**Pull request merged**